From 68eed85162b14dc207e5f4cfa4667ff03158c834 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v6-11.14-68eed85162b-20220525T100743Z-avarab@gmail.com>
In-Reply-To: <cover-v6-00.14-00000000000-20220525T100743Z-avarab@gmail.com>
References: <RFC-cover-v5-00.10-00000000000-20220421T183001Z-avarab@gmail.com>
	<cover-v6-00.14-00000000000-20220525T100743Z-avarab@gmail.com>
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Sat, 21 May 2022 23:16:42 +0200
Subject: [PATCH v6 11/14] ci(github): mention where the full logs can be found
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The full logs are contained in the `failed-tests-*.zip` artifacts that
are attached to the failed CI run. Since this is not immediately
obvious to the well-disposed reader, let's mention it explicitly.

Suggested-by: Victoria Dye <vdye@github.com>
Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 ci/print-test-failures.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ci/print-test-failures.sh b/ci/print-test-failures.sh
index f45e36bdefc..72fae2b0cc4 100755
--- a/ci/print-test-failures.sh
+++ b/ci/print-test-failures.sh
@@ -65,6 +65,7 @@ do
 		if test -n "$do_markup"
 		then
 			printf "\\e[33m\\e[1m=== Failed test: ${TEST_NAME} ===\\e[m\\n"
+			echo "The full logs are in the artifacts attached to this run."
 			cat "t/test-results/$TEST_MARKUP"
 		else
 			echo "------------------------------------------------------------------------"
-- 
2.36.1.1045.gf356b5617dd

